pathgen/pathgen.go

```diff
@@ -681,6 +689,14 @@ type {{ .TypeName }}{{ .WildcardSuffix }} struct {
 	{{- .ExtraWildcardFields }}
 }
 {{- end }}
+
+{{- if and $.PathOriginName (ne $.PathOriginName "openconfig") }}
```

Comment:
```suggestion
```

With the suggestion that I left on the other PR -- consider just generating this method every time. 

We can then make the `"openconfig"` value be set in the code below.